csharp/src/Ice/OutgoingFrame.cs

```diff
@@ -0,0 +1,202 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+
+namespace ZeroC.Ice
+{
+    /// <summary>Indicates the result of the <see cref="OutgoingFrame.CompressPayload"/> operation.</summary>
+    public enum CompressionResult
+    {
+        /// <summary>The encaps payload was successfully compressed.</summary>
```

Comment:
Spell out encapsulation is doc-comments, except here it should be simply "The payload was compressed successfully".